botanique/lib/state/web_socket_bloc.dart

```diff
@@ -85,7 +85,6 @@ class WebSocketBloc extends Bloc<BaseEvent, ServerEvent> {
     add(ClientWantsAllCollections(
         jwt:
             jwt!)); // result from this is handled in main, because it requires interaction with the AllPlantsCubit, where the plants are requested subsequently,
```

Comment:
could we send all this data every time the client send a request of any type instead of requesting it? or is that overkill?